src/core/link-to-dfn.js

```diff
@@ -24,13 +25,10 @@ export function run(conf, doc, cb) {
         if (oldIsDfn && newIsDfn) {
           // Only complain if the user provides 2 <dfn>s
           // for the same term.
-          pub(
-            "error",
-            "Duplicate definition of '" +
-              (dfn_for ? dfn_for + "/" : "") +
-              title +
-              "'"
-          );
+          if (dfn.attr("id") === undefined) {
+            dfn.makeID(null, title);
+          }
+          listOfDuplicateDfns.push(dfn);
```

Comment:
Here, unJQuery it... as a general note, we are trying to remove jQuery. 

```JS
listOfDuplicateDfns.push(dfn[0]);
```

Reply:
Yep, I understand this but as `dfn` was in `jquery` I proceeded further on this.

Reply:
Thats very true. But eventually we will remove jQuery (🤞), so we want any new code to not rely on it. Make sense?

Reply:
Yes makes sense :smiley: